src/MgvOfferMaking.sol

```diff
@@ -323,7 +323,8 @@ contract MgvOfferMaking is MgvHasOffers {
         // bool updateLocal = tick.strictlyBetter(ofp.local.bestTick().strictlyBetter(tick)
         bool shouldUpdateBranch = !insertionTick.strictlyBetter(ofp.local.bestTick());
 
-        ofp.local = dislodgeOffer(offerList, ofp.olKey.tickScale, ofp.oldOffer, ofp.local, shouldUpdateBranch);
+        ofp.local =
+          dislodgeOffer(offerList, ofp.olKey.tickScale, ofp.oldOffer, ofp.local, cachedLocalTick, shouldUpdateBranch);
```

Comment:
At this point `cachedLocalTick == ofp.local.bestTick()`, so it's not strictly necessary to pass `bestTick` separately.

Reply:
I didn't understand that

Reply:
I was just arguing that we don't __need__ to pass `bestTick` to `dislodgeOffer` as mentioned in the comment above :-)